### ISSUE TITLE
Class overwrite when using internal links with *scGenericLink and *scRouterLink

### ISSUE THREAD

**Issue by jcoronel94** [Sitecore/jss#543]
## Description
Hi, 
Similar to #378 , classes are being overwritten when using the *scGenericLink and *scRouterLink directives with internal links. According the angular sample app, these directives are alternatives to the *scLink directive.

## Expected behavior
Classes should preserve and if a class is set on internalLink field then it should append.

## Steps To Reproduce
1. Start application in connected mode
2. Try to render a internal link with class set on the html and  with no class set on the internalLink Field.
`<a class="foo" *scGenericLink="rendering.fields.internalLink"></a>`
`<a class="foo" *scRouterLink="rendering.fields.internalLink"></a>`

## Your Environment

* Sitecore Version:9.3
* JSS Version:15.0.1
* Browser Name and version: All
* Operating System and version (desktop or mobile): All

## Screenshots

![image](https://user-images.githubusercontent.com/15075730/106050833-a2b43c00-60b5-11eb-94a1-744e9be01cc6.png)

![image](https://user-images.githubusercontent.com/15075730/106050537-4cdf9400-60b5-11eb-8a23-beaa70b0e3b3.png)

![image](https://user-images.githubusercontent.com/15075730/106050957-cd05f980-60b5-11eb-9d0d-6150808b270e.png)

**Comment by NijasHameed** [comment]
@jcoronel94 Did you find any workaround?, it works with *scLink but nit with alternatives

**Comment by andreypelykh** [comment]
@NijasHameed There are two workarounds: 
1st:
```html
<a *scGenericLink="rendering.fields.internalLink; attrs: {class: 'foo'}"></a>
```
2nd:
```html
<a [class]="'foo'" *scGenericLink="rendering.fields.internalLink"></a>
```

**Comment by illiakovalenko** [comment]
Fixed in #1050
